packages/lib/modules/pool/actions/add-liquidity/AddLiquidityProvider.tsx

```diff
@@ -41,7 +41,7 @@ import { useWrapUnderlying } from '../useWrapUnderlying'
 export type UseAddLiquidityResponse = ReturnType<typeof _useAddLiquidity>
 export const AddLiquidityContext = createContext<UseAddLiquidityResponse | null>(null)
 
-export function _useAddLiquidity(urlTxHash?: Hash) {
+export function _useAddLiquidity(urlTxHash?: Hash, relicId?: string) {
```

Comment:
Why do we need this relicId inside the provider if we are not doing anything with it? I would prefer to have related relic logic completely decoupled.